### PR TITLE
Issue/313 unselect images

### PR DESCRIPTION
**Pull request by stephanegigandet**

### CODE REVIEW

File: cgi/product_image_unselect.pl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	exit(0);
+}
+
+my $product_ref = process_image_unselect($code, $id);
+
+my $data = encode_json({ status_code => 0, status => 'status ok', imagefield=>$id });
+
+print STDERR "product_image_unselect - JSON data output: $data\n";
+
+print header() . $data;
```
Comment (hangy) [inline cgi/product_image_unselect.pl]: Would probably output this as HTML. I think this should use `header( -type => 'application/json', -charset => 'utf-8' )` as in 234b5c201e62007619c688ece9e63e824a958add.

File: lib/ProductOpener/Images.pm [hunk 2]
```diff
@@ -961,6 +962,28 @@ sub process_image_crop($$$$$$$$$$) {
...
+	print STDERR "Images.pm - process_image_unselect - id: $id\n";
+			
+	# Update the product image data
+	my $product_ref = retrieve_product($code);
+	defined $product_ref->{images} or $product_ref->{images} = {};
+	if (defined $product_ref->{images}{$id}) {
+		delete $product_ref->{images}{$id};
+	}
+
+	store_product($product_ref, "unselected image $id");
```
Comment (hangy) [inline lib/ProductOpener/Images.pm]: What about making this log entry translatable like 0ebb55c633f6c8cceab2334a692e5a8ae1736104?

File: cgi/product_image_unselect.pl (hunk 1 above)
Comment (stephanegigandet) [inline cgi/product_image_unselect.pl]: Thanks, I'll change it.

File: lib/ProductOpener/Images.pm (hunk 2 above)
Comment (stephanegigandet) [inline lib/ProductOpener/Images.pm]: We could, but then we need to do something else than the $comment = lang($comment) if $comment; as there's the id of the image in the comment. Not sure if it's worth it.

**Pull request merged**